evennia/server/portal/ttype.py

```diff
@@ -119,10 +119,10 @@ def will_ttype(self, option):
             if clientname.startswith("MUDLET"):
                 # supports xterm256 stably since 1.1 (2010?)
                 xterm256 = clientname.split("MUDLET", 1)[1].strip() >= "1.1"
-                self.protocol.protocol_flags["FORCEDENDLINE"] = False
-
-            if clientname.startswith("TINTIN++"):
-                self.protocol.protocol_flags["FORCEDENDLINE"] = True
+                # Mudlet likes GA's on a prompt line for the prompt trigger to match, if it's not wanting NOGOAHEAD.
+                if not self.protocol.protocol_flags["NOGOAHEAD"]:
```

Comment:
Motivation here is just in case mudlet accpets that GA is bad, at some point and just uses the lack of a terminating EOL like most other modern clients... :)